### PR TITLE
Refactor base system chapter

### PR DESCRIPTION
**Pull request by gsanchietti**
Some information where hidden inside the legacy doc.

Modifications:
- merge old access.rst and base_system.rst to base_system_legacy.rst
- link legacy doc inside index
- update reference labels

Preview available at: https://nethserver.docs.nethesis.it/en/base_app/index.html

### CODE REVIEW

**Review by DavidePrincipi — CHANGES_REQUESTED** [review]
The `.po` files names and `.tx/config` need to be changed to reflect the naming refactor.

However, I'd prefer to not change the file names at all, to avoid breaking web links. 

As we do not plan to add new features to Nethgui I agree to create a new page that briefely explains the changes introduced with NS 7.9 and *where* to find the old documentation (effectively, the release notes might be enough for this purpose). 

The old documentation could consist in the build of a past version of the manual: we can create a branch to track it in the RTD builds. By this way we can remove the Nethgui parts from the master/v7 branch completely.

Let's discuss this change further.

**Comment by gsanchietti** [comment]
Links have been restored.
Only the `access.rst` page has been merged inside `base_system.rst` and transifex catalog config has been updated accordingly.

File: administrator-manual/en/base_system.rst [hunk 1]
```diff
@@ -158,13 +202,13 @@ A remote system can connect to a :index:`network service`, which is a software
 running on |product| itself.
 
 Each service has a list of "open" ports on which it answers to connections.
-Connections can be accepted from selected zones. Finer grained control of 
+Connections can be accepted from selected zones. Finer grained control of
```
Comment (filippocarletti) [inline administrator-manual/en/base_system.rst:205]: More fine-grained control

File: administrator-manual/en/base_system.rst [hunk 2]
```diff
@@ -1,25 +1,94 @@
...
-=================
-Base system (old)
-=================
+======================
+The old Server Manager
+======================
 
-.. note:: A new Server Manager based on Cockpit is available. See :ref:`base_system2-section`.
+This chapter describes the :dfn:`old Server Manager` web interface. Since |product| 7.9 it must be
+installed from the :ref:`software-center-section` page as it is not picked up any more by the installation
```
Comment (filippocarletti) [inline administrator-manual/en/base_system.rst:8]: ... as it is not included in the default installation [procedure]

**Review by filippocarletti — CHANGES_REQUESTED** [review]

File: administrator-manual/en/base_system.rst (hunk 1 above)
Comment (gsanchietti) [reply administrator-manual/en/base_system.rst]: Done

File: administrator-manual/en/base_system.rst (hunk 2 above)
Comment (gsanchietti) [reply administrator-manual/en/base_system.rst:8]: I do not understand the change.
Can you please use the suggestion feature?

**Pull request merged**